tests/server/api/api_test.go

```diff
@@ -70,6 +70,7 @@ func (s *serverTestSuite) TestReconnect(c *C) {
 		if name != leader {
 			res, e := http.Get(s.GetConfig().AdvertiseClientUrls + "/pd/api/v1/version")
 			c.Assert(e, IsNil)
+			res.Body.Close()
```

Comment:
Why not `defer`?

Reply:
It's a for loop, closing it directly is better.